src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -133,6 +147,24 @@ func newBundleCommand(fs afero.Fs) *cobra.Command {
 	return command
 }
 
+func processPathname(pathname string) (string, error) {
```

Comment:
Why pathname instead of filename? I think we have consistently used filename everywhere (this goes for every `pathname`)

Reply:
I used pathname because here we want to specify the file's location whereas filename only deals with the file's name.

For consistency with our codebase, I'll go with Filepath :+1: